### DISCUSSION TITLE
How can I port/develop an app for Google Android, Web and Microsoft Appsource and Marketplace App Stores

### DISCUSSION THREAD

**Discussion by MsD201x** [freeplane/freeplane#594] [Ideas]
### Discussed in https://github.com/freeplane/freeplane/discussions/591

<div type='discussions-op-text'>

<sup>Originally posted by **MsD201x** August 25, 2022</sup>
**My Problem**:  Frustrated in my learning Freeplane. Why? On desktop, I am ok. But, if on the web at Google, Microsoft apps marketplace, Freeplane isn't there. Freeplane is not an optional app for me and my Freeplane learning development.
**My Solution**:  I want to develop an app for Google android, web and Microsoft appsource and marketplace app stores.</div>

**Comment by MsD201x** [comment]
Ayoa, iMindMap, iThoughts, MindGenius, MindManager, MindMaple, MindMapper, MindMaster, MindMeister, MindNode, XMind
These are the current free but limited mindmap tools available but are not Freeplane.

**Reply by lepczynski** [reply]
The one I was using for a decade or more  before I discovered Freeplane (2024.07) was MindMup.
That is integrated as a free add-on to google drive.
But it's rudimentary in comparison to FP, proprietary, you don't know what's happening with your data, cannot set keybindings, script your functionalities, edit styles.. very basic..

**Comment by ldmpub** [comment]
You're welcomed 👌 if you want to develop a mobile (phone) application version of freeplane.
The task is hard and has been discussed in one of our Freeplane community online conference, read the minute there:  https://github.com/freeplane/freeplane/discussions/339#discussioncomment-2842778

**Reply by Febtw** [reply]
Can Web-FreePlane start from displaying .mm file online? I think if needed, it will be promoted.

**Reply by ldmpub** [reply]
@Febtw What do you mean by `Web-FreePlane`? Yet no web version of Freeplane exists.

**Reply by Febtw** [reply]
Yes, I mean the web version of Freeplane by Web-FreePlane. I want to selfhost freeplane on server and use it in browser across devices.

**Reply by bepolymathe** [reply]
> Yes, I mean the web version of Freeplane by Web-FreePlane. I want to selfhost freeplane on server and use it in browser across devices.

Sorry but I'm not sure I understand either. Could you clarify the project?

**Reply by lepczynski** [reply]
> @Febtw What do you mean by `Web-FreePlane`? Yet no web version of Freeplane exists.

It seems to me that Febtw meant to initiate existence of one `Web-FreePlane` by developing a read-only viewer version for starters.

**Comment by Shahin-rmz** [comment]
It seems there is no intention to make Android or web app.

**Comment by aauro** [comment]
I think it would be great. Even the possibility to run freeplane in a docker container, so that can be accessed  via browser. I need to stop using this software as cannot install it at work :( sad

**Reply by euu2021** [reply]
Can't you use the portable version, at work?

**Reply by aauro** [reply]
USB not allowed

**Reply by euu2021** [reply]
I used Google Remote Desktop on my Android phone for some time, and it's surprisingly good. I think it's possible to use it directly from the browser, too.

**Reply by euu2021** [reply]
> cannot install it at work

Can't you download the portable version directly in the office pc? You can run the portable version without installing it.
